src/components/Heading/Heading.tsx

```diff
@@ -4,6 +4,7 @@ import {
   ChakraComponent,
   Heading as ChakraHeading,
   useMultiStyleConfig,
+  HeadingProps as ChakraHeadingProps,
```

Comment:
Are all default Chakra component props available in this way?

Reply:
I _think_ so. I can't see why not.